crowbar_framework/app/models/role_object.rb

```diff
@@ -28,24 +28,38 @@ def cluster_roles(roles = RoleObject.all)
     end
   end
 
+  def cluster_remote_roles(roles = RoleObject.all)
+    @cluster_remote_roles ||= begin
```

Comment:
TBH, I don't know. I think I'm used to the `@variable ||= begin` way, but indeed it might not be required.